### PR TITLE
Added contribution section to the docs

### PR DESCRIPTION
**Pull request by JulStraus**
I added a contribution section to the documentation. The section is quite a lot inspired by the one of [`EnergyModelsBase`](https://energymodelsx.github.io/EnergyModelsBase.jl/stable/how-to/contribute/), but I think i removed all references to EMX packages after the copy-paste.

In its core, it is inspired by the one of JuMP.

### CODE REVIEW

File: docs/src/contribute.md
Comment (hellemo) [inline docs/src/contribute.md:26]:
```suggestion
2. incorporate the feature request through a fork of the repository and open a pull request.
```

File: docs/src/contribute.md
Comment (trulsf) [inline docs/src/contribute.md:85]:
```suggestion
Please remain patient as it may take potentially some time before we can respond to the changes, although we try to answer as fast as possible.
```

File: docs/src/contribute.md
Comment (trulsf) [inline docs/src/contribute.md:7]:
```suggestion
One approach to contributing to `TimeStruct` is through filing a bug report as an *[issue](https://github.com/sintefore/TimeStruct.jl/issues/new)* when unexpected behaviour is occuring.
```

**Review by trulsf — APPROVED** [review]

**Comment by JulStraus** [comment]
I added all changes. From my end, it is good to merge. I leave it to you @trulsf  to merge it once @hellemo has approved as well (if you consider it required).

A side note:
We moved to EMX to test x64 LTS and latest now on both Windows and Linux while we removed nightly and x86 (it does not really makes sense as the first x86-64 CPUs came out in 2003). We could consider including ARM tests, if supported. This requires adjustments to the Julia Action versions as can be seen in [this PR of EMRP](https://github.com/EnergyModelsX/EnergyModelsRenewableProducers.jl/pull/27/files).

**Review by hellemo — APPROVED** [review]

**Pull request merged**
